.github/workflows/test.yml

```diff
@@ -8,24 +8,33 @@ on:
 
 jobs:
   build:
+    strategy:
```

Comment:
⬆️ - "Run linter" doesn't entirely describe this workflow - it actually builds the project to, and, would presumably and run tests too once such things existed.